sync3/handler/connstate.go

```diff
@@ -99,9 +100,7 @@ func (s *ConnState) load(ctx context.Context, req *sync3.Request) error {
 	if err != nil {
 		return err
 	}
-	for roomID, nid := range loadPositions {
-		s.loadPositions[roomID] = nid
-	}
+	s.loadPositions = loadPositions
```

Comment:
An integration test is failing. I wonder if `loadPositions` is sometimes nil even when `err` is not nil?